testsuites/CBLTester/CBL_Functional_tests/SuiteSetup_FunctionalTests/test_query.py

```diff
@@ -1578,3 +1578,37 @@ def test_query_arthimetic(params_from_base_suite_setup):
 
     qy = Query(base_url)
     qy.query_arthimetic(cbl_db)
+
+
+def test_live_query_response_delay_time(params_from_base_suite_setup):
+    """
+    note: this test case is for android/java only
+    1. call /query_getLiveQueryResponseTime to get delay timer
+    2. validate delay timer is less than 200 millisecond
+    """
+    base_url = params_from_base_suite_setup["base_url"]
+    db = params_from_base_suite_setup["suite_db"]
+    cbl_db = params_from_base_suite_setup["suite_source_db"]
+    liteserv_platform = params_from_base_suite_setup["liteserv_platform"]
+
+    if liteserv_platform in ["ios", "net-msft", "net-uwp", "xamarin-ios", "xamarin-android"]:
```

Comment:
why not this test for other platforms . is support for live query needs to add on other platforms?

Reply:
this is for android only, Blake added live query delaying improvement to android